<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info</class>
 <widget class="QDialog" name="info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color: rgb(93, 0, 0);
font: 25 italic 14pt &quot;Bookman Old Style&quot;;</string>
         </property>
         <property name="text">
          <string>文件大小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">color: rgb(93, 0, 0);
font: 25 italic 14pt &quot;Bookman Old Style&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(93, 0, 0);
font: 25 italic 14pt &quot;Bookman Old Style&quot;;</string>
         </property>
         <property name="text">
          <string>创建时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(93, 0, 0);
font: 25 italic 14pt &quot;Bookman Old Style&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(93, 0, 0);
font: 25 italic 14pt &quot;Bookman Old Style&quot;;</string>
         </property>
         <property name="text">
          <string>修改时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(93, 0, 0);
font: 25 italic 14pt &quot;Bookman Old Style&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
